server/lib/models/model.go

```diff
@@ -3,55 +3,50 @@ package models
 import (
 	"log"
 	"fmt"
+	"strings"
 	"database/sql"
 
-	proto "../../proto"
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/utils"
 
-	_ "github.com/cockroachdb/cockroach/sql/driver"
+	_ "github.com/cockroachdb/pq"
```

Comment:
__[gometalinter]__
* [golint] a blank import should be only in a main or test package, or have a comment justifying it